benchmarks/functional_autograd_benchmark/functional_autograd_benchmark.py

```diff
@@ -134,6 +209,33 @@ def noop():
 
     return elapsed
 
+def run_model_functorch(model_name: str, model_getter: GetterType, args: Any, task: str) -> List[float]:
```

Comment:
Can we deduplicate code between run_model and run_model_functorch? Maybe by passing `run_once` as an argument to `run_model`?